src/workflows/xtrigger/workflow_state/downstream/flow.cylc

```diff
@@ -3,8 +3,7 @@
 [scheduling]
     initial cycle point = 2010
     [[xtriggers]]
-         upstream = workflow_state(workflow=up, task=foo, point=%(point)s, \
-            message='data ready'):PT10S
+         upstream = workflow_state(up//%(point)s/foo:x):PT10S
```

Comment:
It would be helpful for https://github.com/cylc/cylc-flow/issues/5588 if we quoted xtrigger arguments in the documentation:

```suggestion
         upstream = workflow_state("up//%(point)s/foo:x"):PT10S
```